addons/backpacks/functions/fnc_playerVehicleChanged.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+private ["_unit", "_holder"];
+
+_unit = _this select 0;
+_holder = _unit getVariable "ACE_Backpack" select 0;
```

Comment:
No guaranteed that this variable is set.  _holder will be nil
probably want `getVariable ["ACE_Backpack", [objNull, false]]`